geomstats/hyperbolic_space.py

```diff
@@ -161,21 +162,6 @@ def squared_norm(self, vector, base_point=None):
         sq_norm = self.embedding_metric.squared_norm(vector)
         return sq_norm
 
-    def projection_to_tangent_space(self, vector, base_point):
```

Comment:
where is that?

Reply:
Above! this version was copy-pasted (as I told you tuesday) bcs of the problem with objects __init__s.

Reply:
oh right nice! thanks for the cleanup